Formula/bazarr.rb

```diff
@@ -0,0 +1,95 @@
+class Bazarr < Formula
+  include Language::Python::Virtualenv
+
+  desc "Companion to Sonarr and Radarr for managing and downloading subtitles"
+  homepage "https://www.bazarr.media"
+  license "GPL-3.0-only"
+  head "https://github.com/morpheus65535/bazarr.git", branch: "master"
+
+  stable do
+    url "https://github.com/morpheus65535/bazarr/releases/download/v1.0.3/bazarr.zip"
+    sha256 "540a5c4d80110ba35a566c17829cd6f31444db7ade7c6b9d0603eccaaed8a628"
+
+    # Add support for `NO_UPDATE` env var. Remove after next release.
+    patch do
+      url "https://github.com/morpheus65535/bazarr/commit/72aab1e1b4995957d2234b549578d1bf3f169be4.patch?full_index=1"
+      sha256 "e11a2676b5cd9721881c71d6add25039dc2bfad1f9ac658a523e3fed54d4d388"
+    end
+  end
+
+  depends_on "ffmpeg"
+  depends_on "gcc"
+  depends_on "numpy"
+  depends_on "python@3.9"
+
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
+  uses_from_macos "zlib"
+
+  resource "lxml" do
+    url "https://files.pythonhosted.org/packages/3b/94/e2b1b3bad91d15526c7e38918795883cee18b93f6785ea8ecf13f8ffa01e/lxml-4.8.0.tar.gz"
+    sha256 "f63f62fc60e6228a4ca9abae28228f35e1bd3ce675013d1dfb828688d50c6e23"
+  end
+
+  resource "webrtcvad-wheels" do
+    url "https://files.pythonhosted.org/packages/1c/37/56ddf05b6eaf4023b2d3eb069954fd9150b452ee326d0ea20af1d0d4b0c2/webrtcvad-wheels-2.0.10.post2.tar.gz"
+    sha256 "151bf3998fb731afff90dba77808326235370a6bb467a2d1b81345b10d1de10d"
+  end
+
+  def install
+    # Fix "ld: file not found: /usr/lib/system/libsystem_darwin.dylib" for lxml
+    ENV["SDKROOT"] = MacOS.sdk_path if MacOS.version == :sierra
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/Language::Python.site_packages("python3")
+    venv = virtualenv_create(libexec, "python3")
+
+    venv.pip_install resources
+
+    # Stop program from automatically downloading its own binaries.
+    binaries_file = buildpath/"bazarr/binaries.json"
+    rm binaries_file
+    binaries_file.write "[]"
```

Comment:
```suggestion
    (buildpath/"bazarr/binaries.json").write "[]"
```

Reply:
I'm now getting the following error on CI, which may be related to this change?

```
Error: An exception occurred within a child process:
[518](https://github.com/Homebrew/homebrew-core/runs/6326383238?check_suite_focus=true#step:6:518)
    RuntimeError: Will not overwrite /private/tmp/bazarr-20220506-22264-15jii7d/bazarr/binaries.json
```

Reply:
Yeah, reverting this fixes the issue.